client/app/assets/css/redash.css

```diff
@@ -667,3 +667,36 @@ div.table-name:hover {
 .m-2{
   margin:2px;
 }
+
+/* Compare Query Version container */
+/* Offers slight visual improvement (alignment) to modern UAs */
```

Comment:
You can place this along side the component. [Example](https://github.com/getredash/redash/blob/master/client/app/components/app-header/index.js#L5).